rllib/models/torch/torch_distributions.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, *args, **kwargs):
     @abc.abstractmethod
     def _get_torch_distribution(
         self, *args, **kwargs
-    ) -> torch.distributions.Distribution:
+    ) -> "torch.distributions.Distribution":
```

Comment:
same here -- did we need to introduce this in this pr?

Reply:
yes yes. Read above.